src/kernel.zig

```diff
@@ -1,8 +1,13 @@
-const keyboard = @import("./tty/keyboard.zig");
-const printk = @import("./tty/tty.zig").printk;
-const shell = @import("./shell.zig").shell;
+const tty = @import("./tty/tty.zig");
+const DefaultShell = @import("shell/default/shell.zig");
+const printk = tty.printk;
 
 pub fn main() void {
     printk("hello, \x1b[32m{d}\x1b[0m\n", .{42});
```

Comment:
the motd would be better in the default/shell

Reply:
Done. 411a3d6

I added an "on_init" hook, called during initialization.